rally_ovs/plugins/ovs/ovsclients_impl.py

```diff
@@ -67,11 +67,11 @@ def run(self, cmd, opts=[], args=[], stdout=sys.stdout, stderr=sys.stderr):
                 cmd = itertools.chain(cmd_prefix, ["ovn-nbctl"], opts, [cmd], args)
                 self.cmds.append(" ".join(cmd))
 
-            self.ssh.run("\n".join(self.cmds),
-                         stdout=stdout, stderr=stderr)
+            exit_status, data = self.ssh.run(
```

Comment:
this is not necessary, since stdout already contains the output. If we do this, we need to update the requirement.txt to ensure the latest version of rally is used so that "data" is returned from self.ssh.run().

Reply:
@hzhou8 So stdout was returning None. That is the reason this is needed

Reply:
for stdout to return the output, you need to provider the parameter. e.g.:
my_stdout = StringIO()
self.run(stdout=my_stdout)